web_generator/lib/src/translator.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:js_interop';
```

Comment:
It looks like Dart CI wants this file formatted FYI.

Reply:
Updated.